host/host.go

```diff
@@ -52,6 +54,7 @@ func New(log zerolog.Logger, address string, port uint, options ...func(*Config)
 
 	opts := []libp2p.Option{
 		libp2p.ListenAddrStrings(addresses...),
+		libp2p.Transport(webrtc.New),
```

Comment:
Nitpick but I would slide this a line down, after `DefaultTransports`.

Reply:
https://github.com/blocklessnetwork/b7s/pull/168/commits/c4eca09c350bcc9a7f967c11d08144d955a3811b